spec.html

```diff
@@ -16870,7 +16883,7 @@ <h1>Static Semantics: Early Errors</h1>
           DecimalIntegerLiteral :: NonOctalDecimalIntegerLiteral
         </emu-grammar>
         <ul>
-          <li>It is a Syntax Error if the source text matched by this production is strict mode code.</li>
+          <li>It is a Syntax Error if IsStrict(this phrase) is *true*.</li>
```

Comment:
I don't love "this phrase". We could split it, but I also don't love the duplication. What's wrong with "this production"?

Reply:
> I don't love "this phrase".

I don't either, but it has a tiny bit of precedent.

> We could split it, but I also don't love the duplication.

But it's a pretty small amount of duplication.

> What's wrong with "this production"?
> 
> **edit:** I guess the argument could be made that there's a category error there in that "this production" is the grammar production itself, not the parse node resulting from applying this grammar production,

Right. As I say above, it reads like asking if the production itself is strict. (The status quo wording doesn't have that problem.)

> but "this phrase" made that same error IMO,

Well, "this phrase" isn't as well-defined as "this production", so whether it's the same error or a different error or acceptable is a matter of taste. 

> and I really don't think anyone is going to mind.

I would.

----
If you're considering alternatives, we could say:
- this Parse Node
- the current Parse Node
- the subject Parse Node

I'd probably revert the last commit and make that a separate PR though.

Reply:
"this Parse Node" works for me.

Reply:
Marking this as resolved because the change in question has been dropped. (And #3210 changes "this production" to "this Parse Node".)